data/discovery/registry/national-health-service.yaml

```diff
@@ -0,0 +1 @@
+registry: national-health-service
```

Comment:
It needs to be something from here I believe - https://github.com/openregister/openregister-java/blob/master/src/main/resources/config/public-bodies.yaml